en/python_installation/instructions.md

```diff
@@ -2,14 +2,14 @@
 
 > This section is based on a tutorial by Geek Girls Carrots (https://github.com/ggcarrots/django-carrots)
 
-Django is written in Python. We need Python to do anything in Django. Let's start by installing it! We want you to install Python 3.6, so if you have any earlier version, you will need to upgrade it.
+Django is written in Python. We need Python to do anything in Django. Let's start by installing it! We want you to install the latest version of Python 3, so if you have any earlier version, you will need to upgrade it, but if you already have version 3.4 to 3.7 you should be fine.
```

Comment:
Not sure we should mention 3.7 here. How about 3.4 or newer? That's also what the requirements on PyPI are.

Reply:
@ekohl current English version installs Django 2.0.6. Since 2.0 Django supports Python 3.7, so we should be fine here.

Reply:
The requirements for Django 2.x are here:
https://docs.djangoproject.com/en/2.1/releases/2.0/

It says specifically 3.4 to 3.7, at least currently. So I think we should leave it being specific, until Django changes its requirements?